src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -214,6 +215,54 @@ public function getConfigTreeBuilder()
         return $treeBuilder;
     }
 
+    private function addRoutingLoaderSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('routing')
```

Comment:
this can be changed in symfony's settings

Reply:
Are you sure you understand the essence of the changes?

I added this setting to manage Api Platform routing, not Symphony's global routing. Do you have any idea that your bundle can be used together and as an add-on? Why do I need to change the configured and running Symphony application when I need to change the settings of the Api Platform?